terraform/terraform.tfvars

```diff
@@ -6,10 +6,10 @@
 // These are the only settings you need to get started.
 
 // AWS region in which to deploy the BinaryAlert components.
-aws_region = "us-east-1"
+aws_region = "eu-west-1"
```

Comment:
You accidentally committed your config :)

Reply:
Ah sorry, changed